ocs_ci/ocs/resources/pvc.py

```diff
@@ -115,6 +115,16 @@ def backed_sc(self):
         """
         return self.data.get("spec").get("storageClassName")
 
+    @property
+    def get_pvc_vol_mode(self):
+        """
+        Function to get pvc volume_mode
+
+        Returns:
+            (str): The volumeMode Value of pvc_obj
```

Comment:
```suggestion
            str: The volumeMode Value of pvc_obj
```
there should be one empty line after last section

Reply:
done